frontend/src/routes/(authenticated)/admin/+page.ts

```diff
@@ -3,43 +3,70 @@ import { getClient, graphql } from '$lib/gql';
 import type { PageLoadEvent } from './$types';
 import { isAdmin, type LexAuthUser } from '$lib/user';
 import { redirect } from '@sveltejs/kit';
-import { getBoolSearchParam } from '$lib/util/query-params';
-import type { $OpResult, ChangeUserAccountByAdminInput, ChangeUserAccountByAdminMutation, ProjectType } from '$lib/gql/types';
-import type {LoadAdminDashboardQuery} from '$lib/gql/types';
+import { getBoolSearchParam, getSearchParam } from '$lib/util/query-params';
+import type { $OpResult, ChangeUserAccountByAdminInput, ChangeUserAccountByAdminMutation, ProjectFilterInput, ProjectType } from '$lib/gql/types';
+import type { LoadAdminDashboardProjectsQuery, LoadAdminDashboardUsersQuery } from '$lib/gql/types';
 
-export type AdminSearchParams = {
+export const _FILTER_PAGE_SIZE = 100;
+
+export type AdminProjectSearchParams = {
   showDeletedProjects: boolean,
   projectType: ProjectType | undefined,
   userEmail: string | undefined,
   projectSearch: string,
 };
 
-export type Project = LoadAdminDashboardQuery['projects'][number];
-export type User = LoadAdminDashboardQuery['users']['items'][number];
+export type AdminUserSearchParams = {
+  userSearch: string,
+  userEmail: string | undefined,
+}
+
+export type AdminSearchParams = AdminProjectSearchParams & AdminUserSearchParams;
```

Comment:
it looks like we don't need these to be separate types now correct? they aren't being used anywhere else.

Reply:
Ah yeah. I thought I fixed that 🤔.
I'll push that change to develop.